### PR TITLE
style: storybook theming

### PR DESCRIPTION
**Pull request by annsch**
Proposal for a more DB specific UI:
Bildschirmfoto 2022-08-25 um 10 06 01

### CODE REVIEW

**Review by nmerget — APPROVED** [review]

**Comment by nmerget** [comment]
LGTM

**Comment by mfranzke** [comment]
@annsch please have a look in the internal Git system if you could adapt something from the merge request 224 to this one.

**Review by mfranzke — APPROVED** [review]

File: .storybook/dbTheme.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { create } from '@storybook/theming/create';
+
+const TEXT = '#131821';
+const GREY = '#FDFDFD';
+
+export default create({
+  base: 'light',
+  brandTitle: 'DB UI Elements ' + process.env.VERSION,
```
Comment (mfranzke) [inline .storybook/dbTheme.js:8]: this either doesn't seem to work on https://db-ui.github.io/elements/review/feat-storybook-theming/?path=/story/00-intro-welcome--page so far, or just hasn't been deployed so far.

File: .storybook/dbTheme.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { create } from '@storybook/theming/create';
+
+const TEXT = '#131821';
+const GREY = '#FDFDFD';
```
Comment (mfranzke) [inline .storybook/dbTheme.js:4]: maybe we could reference the existing tokens directly from `node_modules/@db-ui/base/build/js/tokens.js`

Comment (annsch) [reply .storybook/dbTheme.js:4]: good point, I'll try this

**Pull request closed without merging**